src/xo-app/backup/new/index.js

```diff
@@ -389,7 +390,11 @@ export default class New extends Component {
           collection: {
             type: 'fetchObjects',
             pattern: {
-              $pool: notPools ? { __not: { __or: pools } } : { __or: pools },
+              $pool: pools && pools.length
+                ? (notPools
+                  ? { __not: { __or: pools } }
+                  : { __or: pools })
```

Comment:
Line of the closing brace should have the same indentation of the line of the opening brace.